pkgs/tools/system/netdata/default.nix

```diff
@@ -94,6 +95,12 @@ stdenv.mkDerivation rec {
        $out/libexec/netdata/plugins.d/slabinfo.plugin.org
     mv $out/libexec/netdata/plugins.d/debugfs.plugin \
        $out/libexec/netdata/plugins.d/debugfs.plugin.org
+    ${lib.optionalString withNdsudo ''
+      mv $out/libexec/netdata/plugins.d/ndsudo \
+         $out/libexec/netdata/plugins.d/ndsudo.org
+
+      ln -s /run/netdata/ndsudo/ndsudo $out/libexec/netdata/plugins.d/ndsudo
```

Comment:
Because most plugins use[ hardcoded path to `ndsudo`](https://github.com/search?q=repo%3Anetdata%2Fnetdata+%22ndsudoPath+%3A%3D%22&type=code) I think it's easier to have a "stub" `ndsudo` in the package itself instead of trying to patch those plugins to use other location. 
What do you think?